test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -663,7 +698,6 @@ class ExtAuthzHttpIntegrationTest : public HttpIntegrationTest,
     server_uri:
       uri: "ext_authz:9000"
       cluster: "ext_authz"
-      timeout: 300s
```

Comment:
This timeout is for HTTP mode. It's not related to the timeout in the gRPC configuration.

Reply:
OH, you are testing HTTP mode timeout as well.